src/hooks/useENS.ts

```diff
@@ -0,0 +1,37 @@
+import { keccak_256 as sha3 } from 'js-sha3';
+import { useCallback } from 'react';
+import {
+  getBaseRegistrarImplementationContract,
+  getENSContract,
+  getENSRegistrarContract,
+} from '@rainbow-me/helpers/ens';
+export default function useENS() {
```

Comment:
This probably doesn't need to be a hook. Couldn't we put this functionality in `helpers/ens.ts`?

e.g.

```tsx
const ensContract = new Contract(...);
const ensRegistrarContract = new Contract(...);
const baseRegistrarImplementationContract = new Contract(...)

export const getResolver = async (name: string) => ensContract.resolver(name);
export const getAvailable = async (name: string) => ensRegistrarContract.available(name);
export const getNameExpires = async (name: string) => ensBaseRegistrarContract.nameExpires('0x' + sha3(name));
```

Reply:
moved it out! i maintained the hook since we'll need to start interacting with state stuff